### PR TITLE
Lean 3 and mathlib3

### PR DESCRIPTION
**Pull request by madvorak**

### CODE REVIEW

**Comment by PatrickMassot** [comment]
I don't think we should change `mathlib` to `mathlib3` since this name isn't used anywhere else.

**Comment by madvorak** [comment]
> I don't think we should change `mathlib` to `mathlib3` since this name isn't used anywhere else.

ah sry

File: templates/install/windows.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Installing Lean and mathlib on Windows
```
Comment (PatrickMassot) [inline templates/install/windows.md:1]: Could you also write Lean 3 here?

File: templates/install/windows.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ This document describes using VS Code (for emacs, look at https://github.com/lea
    (or ![(image of icon)](img/extensions-icon.png) in older versions) in the side bar on the left edge of
    the screen (or press <kbd>Shift</kbd><kbd>Ctrl</kbd><kbd>X</kbd>) and search for `leanprover`.
 4. Select the `lean` extension (unique name `jroesch.lean`). There is also a
-   `lean4` extension, but that one does not work with mathlib.
+   `lean4` extension, but that one would require us to work with mathlib4.
```
Comment (PatrickMassot) [inline templates/install/windows.md:83]: That sentence is a bit weird. Maybe 
```suggestion
   `lean4` extension, but that one isn't relevant here.
```

**Pull request merged**